src/SIL.XForge.Scripture/ClientApp/e2e/src/login.po.ts

```diff
@@ -29,8 +29,14 @@ export class LoginPage {
     await this.loginButton.click();
 
     // wait for redirect before enabling angular wait
-    await browser.sleep(300);
-    browser.waitForAngularEnabled(true);
+    //await browser.sleep(300);
+    // browser.waitForAngularEnabled(true);
```

Comment:
The two comment code lines can be deleted.

Reply:
Okay sure, we removed two commented lines. Thank you.